firebase-config/CHANGELOG.md

```diff
@@ -0,0 +1,332 @@
+# Unreleased
+
+# 21.1.2
+* [changed] Updated dependency of `play-services-basement` to its latest
+  version (v18.1.0).
+
+## Kotlin
+The Kotlin extensions library transitively includes the updated
+`firebase-config` library. The Kotlin extensions library has no additional
+updates.
+
+# 21.1.1
+* [fixed] Fixed a bug that caused HTTP errors in some locales. For more
+  information, see
+  <a href="https://github.com/firebase/firebase-android-sdk/issues/3757"
+     class="external">GitHub Issue #3757</a>
```

Comment:
Looks like here we have html instead of markdown?

Reply:
Yes. The original markdown source has the content like this. While we are going to provide the ability of just use (#XXX) as format, teams can use html if they want to. This is a valid use of [html in markdown](https://daringfireball.net/projects/markdown/syntax#html).